sample-info/diagnosis-groupings.tsv

```diff
@@ -0,0 +1,54 @@
+ontology_id	human_readable_value	submitted_diagnosis	diagnosis_group
+MONDO:0016684	Anaplastic astrocytoma	Anaplastic astrocytoma	Brain and CNS
```

Comment:
Looking at these diagnosis groups, I was wondering if we wanted to use MONDO terms here too (maybe not for display, but we could include them in the table)? My general thought is that if we can use ontology-based groups, we can more easily justify them.

We could use:
- [`MONDO:0021248`: nervous system neoplasm](http://purl.obolibrary.org/obo/MONDO_0021248) for `Brain and CNS`
- [`MONDO:0002334`: hematopoietic and lymphoid system neoplasm](http://purl.obolibrary.org/obo/MONDO_0002334) for `leukemia`
- [`MONDO:0044334`: `connective and soft tissue neoplasm`](http://purl.obolibrary.org/obo/MONDO_0044334) for sarcomas
And then leave an "other" category.

This would change a few categories, but I am had questions about some of them them anyway: 
- is there a reason you don't have neuroblastoma and retinoblastoma in the Brain and CNS category?
- in the ontology, rhabdoid tumor falls under `connective and soft tissue neoplasm`; I'm not sure if we would want to maintain that...

This makes the categories more unbalanced, which could create a different problem... we might have to revert to making separate bar plots (though I would try using `patchwork` to join them, as I think you can get easier control over placement). 

All of this I might say to push off to a separate PR. The current version strikes me as _good_, so the real question is whether we want to use Ontology-based groupings, and then we might have consequences that flow from that.

Reply:
I don't know that we are planning to include that table that's in the repo in the paper, but we probably should. If we do, then I think having the MONDO term would be a good idea so I'm okay with this change. 

> is there a reason you don't have neuroblastoma and retinoblastoma in the Brain and CNS category?

I was on the fence about Neuroblastoma because most of the cases are peripheral and found in abdomen, adrenal gland, pelvis, etc. There is a separate classification that is CNS Neuroblastoma, which is why I didn't originally include it. But looking at the parent terms for Neuroblastoma in MONDO, you are probably right. I could see an argument for both ways since I don't think it's really a traditional CNS tumor. 

For retinoblastoma, this is just from me thinking of it as a solid tumor that's different from gliomas, even though I guess technically it is a CNS tumor. 

> in the ontology, rhabdoid tumor falls under connective and soft tissue neoplasm; I'm not sure if we would want to maintain that...

I think I would actually use the MONDO: for `sarcoma` instead of the connective and soft tissue. It's more specific to the fact that we are only including sarcomas and not soft tissue which is why Rhabdoid is separate. 

I think right now I will for sure change over Retinoblastoma, but I won't change the other ones. Unless you feel strongly about Neuroblastoma. And then I'll open an issue for deciding if we want to use the MONDO groups or not.

Reply:
> I think I would actually use the MONDO: for `sarcoma` instead of the connective and soft tissue.

Makes sense. I think I was leaning toward connective & soft tissue because it was more at the same "level", but that was not a strong sentiment!